docs/administrationGuide.md

```diff
@@ -105,14 +106,14 @@ cd rpm
 ./create-rpm.sh <release-number> <version-number>
 ```
 Where `<version-number>` is the version (x.y.z) you want the package to have and `<release-number>` is an increasing
-number dependent on previous installations. 
+number dependent on previous installations.
 
-# <a name="sanity"> Sanity checks </a>
+# Sanity checks
```

Comment:
Now index link to this subsection does not work

Reply:
Fixed - c2bb99a - ToC updated